parser/CMakeLists.txt

```diff
@@ -114,6 +114,7 @@ add_library(quickstep_parser_ParseSubqueryTableReference ParseSubqueryTableRefer
 add_library(quickstep_parser_ParseTableReference ParseTableReference.cpp ParseTableReference.hpp)
 add_library(quickstep_parser_ParseTreeNode ../empty_src.cpp ParseTreeNode.hpp)
 add_library(quickstep_parser_ParserUtil ParserUtil.cpp ParserUtil.hpp)
+add_library(quickstep_parser_ParseWindow ../empty_src.cpp ParseWindow.hpp)
```

Comment:
Please sort in the alphabet order, i.e., putting between `quickstep_parser_ParseTreeNode` and `quickstep_parser_ParserUtil`.